running commands is broken due to missing dependency when using 0.10.0-alpha.0 release

## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
When installing latest `alpha.0` there is an error for a missing dependency; **fs-extra**.
```sh
% npm run develop

> greenwood-getting-started@1.0.0 develop /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started
> greenwood develop

internal/modules/cjs/loader.js:797
    throw err;
    ^

Error: Cannot find module 'fs-extra'
Require stack:
- /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/lifecycles/config.js
- /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/lifecycles/compile.js
- /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/commands/build.js
- /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:794:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/lifecycles/config.js:1:12)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/lifecycles/config.js',
    '/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/lifecycles/compile.js',
    '/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/commands/build.js',
    '/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/index.js'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! greenwood-getting-started@1.0.0 develop: `greenwood develop`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the greenwood-getting-started@1.0.0 develop script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/owenbuckley/.npm/_logs/2020-12-13T16_15_30_792Z-debug.log
```

Temporary work around for now that works is to install `fs-extra` into the project locally.


## Details
Looks like somewhere along the way we were using `fs-extra`
<img width="1357" alt="Screen Shot 2020-12-13 at 11 10 08 AM" src="https://user-images.githubusercontent.com/895923/102017348-6ced0b00-3d34-11eb-8288-05bf33531074.png">


Looks like right now it is just coming from **Lerna**.
```sh
% yarn why fs-extra
yarn why v1.12.3
[1/4] 🤔  Why do we have the module "fs-extra"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "fs-extra@8.1.0"
info Reasons this module exists
   - "_project_#lerna#@lerna#create" depends on it
   - Hoisted from "_project_#lerna#@lerna#create#fs-extra"
   - Hoisted from "_project_#lerna#@lerna#import#fs-extra"
   - Hoisted from "_project_#lerna#@lerna#init#fs-extra"
   - Hoisted from "_project_#lerna#@lerna#publish#fs-extra"
   - Hoisted from "_project_#lerna#@lerna#version#@lerna#conventional-commits#fs-extra"
   - Hoisted from "_project_#lerna#@lerna#bootstrap#@lerna#npm-install#fs-extra"
   - Hoisted from "_project_#lerna#@lerna#publish#@lerna#npm-publish#fs-extra"
   - Hoisted from "_project_#lerna#@lerna#exec#@lerna#profiler#fs-extra"
   - Hoisted from "_project_#lerna#@lerna#bootstrap#@lerna#symlink-binary#fs-extra"
   - Hoisted from "_project_#lerna#@lerna#bootstrap#@lerna#symlink-dependencies#fs-extra"
   - Hoisted from "_project_#lerna#@lerna#bootstrap#@lerna#symlink-binary#@lerna#create-symlink#fs-extra"
   - Hoisted from "_project_#lerna#@lerna#publish#@lerna#pack-directory#@lerna#get-packed#fs-extra"
   - Hoisted from "_project_#lerna#@lerna#bootstrap#@lerna#symlink-dependencies#@lerna#resolve-symlink#fs-extra"
info Disk size without dependencies: "284KB"
info Disk size with unique dependencies: "376KB"
info Disk size with transitive dependencies: "376KB"
info Number of shared dependencies: 3
✨  Done in 0.68s.
```

Normal `fs` should be fine, but it is also fine if it's kept around as just a `devDependency` too.

--- a/packages/cli/src/data/cache.js
+++ b/packages/cli/src/data/cache.js
@@ -1,7 +1,7 @@
 const { ApolloClient } = require('apollo-client');
 const createHttpLink = require('apollo-link-http').createHttpLink;
 const fetch = require('node-fetch');
-const fs = require('fs-extra');
+const fs = require('fs');
 const { gql } = require('apollo-server');
 const InMemoryCache = require('apollo-cache-inmemory').InMemoryCache;
 const path = require('path');

--- a/packages/cli/src/lifecycles/config.js
+++ b/packages/cli/src/lifecycles/config.js
@@ -1,4 +1,4 @@
-const fs = require('fs-extra');
+const fs = require('fs');
 const path = require('path');
 const url = require('url');
 
@@ -25,7 +25,7 @@ module.exports = readAndMergeConfig = async() => {
       // deep clone of default config
       let customConfig = Object.assign({}, defaultConfig);
       
-      if (await fs.exists(path.join(process.cwd(), 'greenwood.config.js'))) {
+      if (fs.existsSync(path.join(process.cwd(), 'greenwood.config.js'))) {
         const userCfgFile = require(path.join(process.cwd(), 'greenwood.config.js'));
         const { workspace, devServer, publicPath, title, markdown, meta } = userCfgFile;
 
@@ -45,7 +45,7 @@ module.exports = readAndMergeConfig = async() => {
             customConfig.workspace = workspace;
           }
 
-          if (!await fs.exists(customConfig.workspace)) {
+          if (!fs.existsSync(customConfig.workspace)) {
             reject('Error: greenwood.config.js workspace doesn\'t exist! \n' +
               'common issues to check might be: \n' +
               '- typo in your workspace directory name, or in greenwood.config.js \n' +

--- a/packages/cli/src/lifecycles/context.js
+++ b/packages/cli/src/lifecycles/context.js
@@ -1,4 +1,4 @@
-const fs = require('fs-extra');
+const fs = require('fs');
 const path = require('path');
 // const defaultTemplatesDir = path.join(__dirname, '../templates/');
 // const defaultConfigDir = path.join(__dirname, '../config');
@@ -81,8 +81,10 @@ module.exports = initContexts = async({ config }) => {
         scratchDir
       };
 
-      if (!await fs.ensureDir(scratchDir)) {
-        await fs.mkdirs(scratchDir);
+      if (!fs.existsSync(scratchDir)) {
+        fs.mkdirSync(scratchDir, {
+          recursive: true
+        });
       }
       
       resolve(context);

--- a/packages/cli/src/lifecycles/graph.js
+++ b/packages/cli/src/lifecycles/graph.js
@@ -1,6 +1,5 @@
 #!/usr/bin/env node
-const fs = require('fs-extra');
-const { promises: fsp } = require('fs');
+const fs = require('fs');
 const crypto = require('crypto'); // TODO pretty heavy just for some hashing?
 const path = require('path');
 const fm = require('front-matter');
@@ -14,8 +13,8 @@ const createGraphFromPages = async (pagesDir, config) => {
       const pagesIndexMap = new Map();
       let pagesIndex = 0;
 
-      const walkDirectory = async(directory) => {
-        let files = await fs.readdir(directory);
+      const walkDirectory = async (directory) => {
+        let files = await fs.promises.readdir(directory);
 
         return Promise.all(files.map((file) => {
           const filenameHash = crypto.createHash('md5').update(`${directory}/${file}`).digest('hex');
@@ -34,7 +33,7 @@ const createGraphFromPages = async (pagesDir, config) => {
             try {
 
               if (isMdFile && !stats.isDirectory()) {
-                const fileContents = await fs.readFile(filePath, 'utf8');
+                const fileContents = await fs.promises.readFile(filePath, 'utf8');
                 const { attributes } = fm(fileContents);
                 let { label, template, title } = attributes;
                 let { meta } = config;
@@ -221,11 +220,7 @@ module.exports = generateGraph = async (compilation) => {
 
       compilation.graph = await createGraphFromPages(context.pagesDir, config);
 
-      if (!fs.existsSync(context.scratchDir)) {
-        await fsp.mkdir(context.scratchDir);
-      }
-      
-      await fsp.writeFile(`${path.join(process.cwd(), '.greenwood')}/graph.json`, JSON.stringify(compilation.graph));
+      await fs.promises.writeFile(`${path.join(process.cwd(), '.greenwood')}/graph.json`, JSON.stringify(compilation.graph));
 
       resolve(compilation);
     } catch (err) {

--- a/packages/cli/test/cases/eject.default/eject.default.spec.js
+++ b/packages/cli/test/cases/eject.default/eject.default.spec.js
@@ -8,7 +8,7 @@
  * User Command
  * greenwood eject
  */
-const fs = require('fs-extra');
+const fs = require('fs');
 const path = require('path');
 const expect = require('chai').expect;
 const TestBed = require('../../../../../test/test-bed');
@@ -58,7 +58,7 @@ xdescribe('Eject Greenwood With: ', function() {
 
       configFiles.forEach(file => {
         if (file !== 'eject.default.spec.js' && file !== 'node_modules') {
-          fs.remove(path.join(__dirname, file));
+          fs.rmdirSync(path.join(__dirname, file));
         }
       });
     });
@@ -116,7 +116,7 @@ xdescribe('Eject Greenwood With: ', function() {
 
       configFiles.forEach(file => {
         if (file !== 'eject.default.spec.js' && file !== 'node_modules') {
-          fs.remove(path.join(__dirname, file));
+          fs.rmdirSync(path.join(__dirname, file));
         }
       });
     });
@@ -140,7 +140,7 @@ xdescribe('Eject Greenwood With: ', function() {
 
     configFiles.forEach(file => {
       if (file !== 'eject.default.spec.js') {
-        fs.remove(path.join(__dirname, file));
+        fs.rmdirSync(path.join(__dirname, file));
       }
     });
   });

--- a/test/test-bed.js
+++ b/test/test-bed.js
@@ -85,8 +85,11 @@ module.exports = class TestBed {
         await Promise.all(setupFiles.map((file) => {
           const dir = path.join(this.rootDir, file.dir.split('/')[0]);
 
-          return fs.promises.rmdir(dir, { recursive: true });
+          return fs.existsSync(dir)
+            ? fs.promises.rmdir(dir, { recursive: true })
+            : Promise.resolve();
         }));
+        
         resolve();
       } catch (err) {
         reject(err);

--- a/test/test-bed.js
+++ b/test/test-bed.js
@@ -6,7 +6,7 @@
  * There are a number of examples in the CLI package you can use as a reference.
  *
  */
-const fs = require('fs').promises;
+const fs = require('fs');
 const os = require('os');
 const path = require('path');
 const { spawn } = require('child_process');
@@ -44,10 +44,10 @@ module.exports = class TestBed {
 
               await new Promise(async(resolve, reject) => {
                 try {
-                  await fs.mkdir(targetDir, {
+                  await fs.promises.mkdir(targetDir, {
                     recursive: true
                   });
-                  await fs.copyFile(targetSrc, targetPath);
+                  await fs.promises.copyFile(targetSrc, targetPath);
                   resolve();
                 } catch (err) {
                   reject(err);
@@ -74,13 +74,18 @@ module.exports = class TestBed {
   teardownTestBed() {
     return new Promise(async(resolve, reject) => {
       try {
-        await fs.rmdir(path.join(this.rootDir, '.greenwood'), { recursive: true });
-        await fs.rmdir(path.join(this.rootDir, 'public'), { recursive: true });
+        if (fs.existsSync(this.buildDir)) {
+          await fs.promises.rmdir(this.buildDir, { recursive: true }); 
+        }
+
+        if (fs.existsSync(this.publicDir)) {
+          await fs.promises.rmdir(this.publicDir, { recursive: true }); 
+        }
 
         await Promise.all(setupFiles.map((file) => {
           const dir = path.join(this.rootDir, file.dir.split('/')[0]);
 
-          return fs.rmdir(dir, { recursive: true });
+          return fs.promises.rmdir(dir, { recursive: true });
         }));
         resolve();
       } catch (err) {

--- a/test/test-bed.js
+++ b/test/test-bed.js
@@ -6,7 +6,7 @@
  * There are a number of examples in the CLI package you can use as a reference.
  *
  */
-const fs = require('fs-extra');
+const fs = require('fs').promises;
 const os = require('os');
 const path = require('path');
 const { spawn } = require('child_process');
@@ -44,8 +44,10 @@ module.exports = class TestBed {
 
               await new Promise(async(resolve, reject) => {
                 try {
-                  await fs.ensureDir(targetDir, { recursive: true });
-                  await fs.copy(targetSrc, targetPath);
+                  await fs.mkdir(targetDir, {
+                    recursive: true
+                  });
+                  await fs.copyFile(targetSrc, targetPath);
                   resolve();
                 } catch (err) {
                   reject(err);
@@ -72,11 +74,13 @@ module.exports = class TestBed {
   teardownTestBed() {
     return new Promise(async(resolve, reject) => {
       try {
-        await fs.remove(path.join(this.rootDir, '.greenwood'));
-        await fs.remove(path.join(this.rootDir, 'public'));
+        await fs.rmdir(path.join(this.rootDir, '.greenwood'), { recursive: true });
+        await fs.rmdir(path.join(this.rootDir, 'public'), { recursive: true });
 
         await Promise.all(setupFiles.map((file) => {
-          return fs.remove(path.join(this.rootDir, file.dir.split('/')[0]));
+          const dir = path.join(this.rootDir, file.dir.split('/')[0]);
+
+          return fs.rmdir(dir, { recursive: true });
         }));
         resolve();
       } catch (err) {

